### PR TITLE
Allow any case/seq to be used for “[CI SKIP]”

### PR DESCRIPTION
**Pull request by elia**
Eg.

[ci skip]
[skipci]
[skip ci]
[CI SKIP]

### CODE REVIEW

File: server/handler/hook.go [hunk 1]
```diff
@@ -43,7 +44,9 @@ func PostHook(c web.C, w http.ResponseWriter, r *http.Request) {
 	// in some cases we have neither a hook nor error. An example
 	// would be GitHub sending a ping request to the URL, in which
 	// case we'll just exit quiely with an 'OK'
-	if hook == nil || strings.Contains(hook.Message, "[CI SKIP]") {
+	const ci_skip = `\[(?i:ci *skip|skip *ci)\]`
```
Comment (Bugagazavr) [inline server/handler/hook.go]: const should be declared outside function, go uses camelCase instead snake_case

**Comment by bradrydzewski** [comment]
agreed, looks great, just need to change to camel case :)

**Comment by elia** [comment]
thanks, changed to camelCase and avoided the constant altogether

**Comment by bradrydzewski** [comment]
great, thanks!

**Pull request merged**